src/main/java/com/karankumar/booksapi/repository/BookRepository.java

```diff
@@ -16,7 +16,13 @@
 package com.karankumar.booksapi.repository;
 
 import com.karankumar.booksapi.model.Book;
+import java.util.List;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
 public interface BookRepository extends CrudRepository<Book, Long> {
+  
+    @Query("SELECT DISTINCT b FROM Book b LEFT JOIN FETCH b.authors")
+    List<Book> findAllBooks();
```

Comment:
Could you please also write a test for this to check whether this works as intended?

Reply:
integration test or unit test

Reply:
Unit test, please. So populate the repository and then check whether it find all of the books